### PR TITLE
Startup fails if when /var/lib/pgsql/data is non-empty

### PR DESCRIPTION
**Pull request by gsaslis**
On OpenSUSE 13.1, startup fails if when /var/lib/pgsql/data is non-empty

* Changed the time when the configuration files are placed into the directory to be after the first start of the server.
 
* Subsequently changed the notifies to :immediately, otherwise changes I'd made to these files would not have been taken into account. I believe this is better than making whoever's using the cookbook/recipe explicitly have to restart the service if they've made any changes to conf files.

### CODE REVIEW

File: recipes/server_conf.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@
   owner "postgres"
   group "postgres"
   mode 0600
-  notifies change_notify, 'service[postgresql]', :delayed
+  notifies change_notify, 'service[postgresql]', :immediately
```
Comment (slyness) [inline recipes/server_conf.rb]: This is a direct change to the current expected behavior. Can you explain a little more why you chose to switch to `:immediately`? (and also at line 33)

Comment (slyness) [inline recipes/server_conf.rb]: Oh. I see you have very descriptive commit messages. I'll look at those more closely and they might answer my current questions.

Comment (gsaslis) [inline recipes/server_conf.rb]: sure thing - please let me know if you need any further explanations, or if you have any doubts...  ; )

**Comment by gsaslis** [comment]
Hi there, 

Any plans to merge this in soon? 
Please just let me know if there's something else I should work on to help get this sorted.

**Comment by luckymike** [comment]
This appears fairly sound to me, but the PR can't be automatically merged. @gsaslis can you address the conflicts and update this PR?

Thanks!

**Comment by gsaslis** [comment]
Thanks @luckymike - should be ready for you to merge now..  ; )

**Comment by gsaslis** [comment]
@luckymike -- is this still a go?

**Comment by slyness** [comment]
I have added this to our to do list for this week. We should be able to address it soon. Thanks for your patience.

**Comment by gsaslis** [comment]
no worries - please feel free to let me know if there's something I can do
to help...  : )

-- 
Yorgos Saslis
Software Engineer

**Comment by gsaslis** [comment]
@slyness - this seems to have gotten lost somewhere in your todo list.. Just bumping this up... ; )

**Comment by slyness** [comment]
finally.

**Pull request merged**